ethereum/api/src/test/resources/org/hyperledger/besu/ethereum/api/jsonrpc/eth/eth_config.json

```diff
@@ -34,9 +33,13 @@
           "BEACON_ROOTS_ADDRESS": "0x000f3df6d732807ef1319fb7b8bb8522d0beac02"
         }
       },
-      "currentHash": "f503e444",
+      "currentHash": "0x3025d871",
+      "currentForkId": "0x3025d871",
       "next": null,
-      "nextHash": null
+      "nextHash": null,
+      "nextForkId": null,
+      "lastHash": "0x3025d871",
+      "lastForkId": "0x3025d871"
```

Comment:
I think we are missing the last section: https://github.com/ethereum/EIPs/pull/9989/files#diff-9d750bc73512839dbf2601b21a92e25a3447cabb610ed59f97148efe6d04c7d7R303